docs/_static/didman/v1.yaml

```diff
@@ -213,6 +213,40 @@ paths:
           description: The service has been removed
         default:
           $ref: '../common/error_response.yaml'
+  /internal/didman/v1/search/organizations:
+    parameters:
+      - name: query
+        in: query
+        description: >
+          Query used for searching the organization by name. The query is matched to the organization's name in a SQL's
+          "LIKE" fashion: it matches partial strings and also names that sound like the given query,
+          using a phonetic transformation algorithm. The exact algorithm depends on the implementation and might change without notice.
```

Comment:
| The exact algorithm depends on the implementation and might change without notice.
that's change management, not a functional description

Reply:
Removed it 👍